### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by johndeere4344** [weissi1994/LethalMod-ESP#5]
I downloaded manually and everything is fine, adjusted config but none of the esp appears in game, I used cmd to see if it loaded and it did. Should I use this mod by itself or does it work with additional mods?

**Comment by weissi1994** [comment]
as shown in https://thunderstore.io/c/lethal-company/p/blob/LethalMod/
BepInEx is the only required depencency.

Which version of the mod are you using?

As a side-note: The ESP itself will only show up when ingame. but in the game-menu you should already see the available keybindings and options in the top left of the screen.

**Comment by kennyhngo** [comment]
Yeah same issue with me. Created new mod profile to make sure no other mods were interfering with it. Installed LethalMod from thunderstore, which installed BepInEx dependency. Using latest mod version 0.0.10, can confirm console logs that the plugin loaded. Pressing the digit keys don't work and I do not see anything new in the top left of screen.

**Comment by weissi1994** [comment]
can you check with v0.0.14, available at: https://thunderstore.io/c/lethal-company/p/blob/LethalMod/versions/
